src/steps/access/converter.ts

```diff
@@ -7,39 +7,73 @@ import {
 } from '@jupiterone/integration-sdk-core';
 
 import { Entities } from '../constants';
-import { AcmeGroup, AcmeUser } from '../../types';
+import { CloudbeesGroup, CloudbeesRole, CloudbeesUser } from '../../types';
 
-export function createUserEntity(user: AcmeUser): Entity {
+export function createUserKey(id: string): string {
+  return `cloudbees_user:${id}`;
+}
+
+export function createGroupKey(id: string): string {
+  return `cloudbees_group:${id}`;
+}
+
+export function createRoleKey(id: string): string {
+  return `cloudbees_role:${id}`;
+}
+
+export function createUserEntity(user: CloudbeesUser): Entity {
   return createIntegrationEntity({
     entityData: {
       source: user,
       assign: {
         _type: Entities.USER._type,
         _class: Entities.USER._class,
-        _key: user.id,
-        username: 'testusername',
-        email: 'test@test.com',
-        active: true, // this is a required property
-        // This is a custom property that is not a part of the data model class
-        // hierarchy. See: https://github.com/JupiterOne/data-model/blob/master/src/schemas/User.json
-        firstName: 'John',
+        _key: createUserKey(user.user.id),
+        name: user.user.fullName,
+        displayName: user.user.fullName,
+        username: user.user.fullName,
+        absoluteUrl: user.user.absoluteUrl,
```

Comment:
absoluteUrl is the perfect candidate for `webLink` since it already contains the full URL path.

```suggestion
        webLink: user.user.absoluteUrl,
```